contents/blog/startup-data-stack.md

```diff
@@ -0,0 +1,108 @@
+
+- The complexity of data and tools requires specialization.
+- Data security, safety, and privacy requirements.
+- Lack of knowledge of data available or how to use it.
+
+For example, engineers might need to write optimized SQL queries to understand customer feature usage. This requires an understanding of SQL, data availability, data structured, as well as access. These all create friction on what engineers want to do: build better products. 
+
+## PostHog’s vision for the future of the startup data stack
+
+PostHog is not content with the "modern data stack." The gaps and complexity it creates limits the ability to [build successful products](/handbook/strategy/overview). To change this, we are improving how startups gather and analyze data at the earliest stages and consolidating more of the stack as they scale.
+
+To make this happen, we are improving PostHog as a [customer data platform (CDP)](https://github.com/PostHog/posthog/issues/13126). We have built the key functionality, such as the ability to:
+
+- Receive data from your app(s) and site(s) with our [SDKs](/docs/integrate?tab=sdks)
+- Receive data from sources like [Stripe](/apps/stripe-connector), [Hubspot](/apps/hubspot-connector), [Intercom](/apps/intercom)
+- Export to destinations such as [Snowflake](/apps/snowflake-export), [BigQuery](/apps/bigquery-export), or [RedShift](/apps/redshift-export)
+
+Improving reliability, integrations, and UX is critical for success here. Being a CDP enables startups to have their data and tools in one place with PostHog. More data also improves the depth of insights engineers can get from PostHog without having to use other tools such as BI or visualization.
+
+![PostHog CDP](../images/blog/startup-data-stack/posthog-cdp.png)
+
+We are also figuring out what PostHog looks like as [a data warehouse](https://github.com/PostHog/posthog/issues/14406) utilizing [ClickHouse](/docs/how-posthog-works/clickhouse). Data accuracy and reporting flexibility matter at scale. Being a warehouse enables PostHog to provide this to startups, and integrate it with tools they are already using PostHog for. With PostHog as a warehouse, startups don’t need to migrate or set up many of the modern data stack tools.
+
+We want to make it easy to evaluate whether products are working or not. The ability for engineers to find and analyze the metrics they want, along with tools like experiments and session recordings, is key to this. The modern data stack creates complexity and silos that make figuring out what to work on and what’s working difficult.
+
+Integrating more of the parts of the startup data stack into PostHog enables engineers to continue to access the data, insights, and tools they need to build a great product. This current data stack evolution becomes large, complicated, and siloed quickly. Building an ideal data stack for engineers is what PostHog aims to do, and by doing so, we'll achieve our goal of more successful products in the world.
```

Comment:
Maybe we can update this section to discuss what's been built at the offsite + HogQL. It feels a bit stronger now we've started building this stuff instead of just talking about it.

Reply:
I've rewritten this, didn't mention HogQL, but did talk about data warehousing (which is more relevant I think).